docs/eq_runner_to_downstream_payload_v2.md

```diff
@@ -33,10 +33,11 @@ In additional to the field above, a schema selection field will be provided whic
 
 One of the following will be present:
```

Comment:
- presumably this is "one of the following _must_ be present" based on the [definition](https://github.com/ONSdigital/ons-schema-definitions/blob/update-docs-for-cir/schemas/launch_v2.json#L74C13-L84) ?
- also based on that definition, will the launch error if more that one is provided, or is there order of precedence that will be honoured?

Reply:
Updated the wording now, thanks 👍. The launch would error as we will update the validation that we have in Runner to enforce that these vars are mutually exclusive, to ensure that we're only passing one of the vars downstream to SDX.